client/merchant-feedback-prompt/negative-modal.tsx

```diff
@@ -70,6 +72,25 @@ export const NegativeFeedbackModal: React.FC< NegativeFeedbackModalProps > = ( {
 					</a>
 					.
 				</p>
+				<p>
```

Comment:
Non-blocking comment - Will it be helpful to add a track event to check how many people actually ended up contacting support from the negative modal?

Reply:
Great idea. Low effort to add, with the potential for valuable insight.

Reply:
Added in fd0e1cb. Tracks event registered in https://github.com/Automattic/tracks-events-registration/pull/2792